frontend/benefit/applicant/.eslintrc.js

```diff
@@ -4,6 +4,7 @@ module.exports = {
   extends: ['auto'],
   rules: {
     'no-secrets/no-secrets': ['error', { tolerance: 4.2 }],
+    'sonarjs/cognitive-complexity': ['error', 20],
```

Comment:
Perhaps it would be better to split the overlong method (usePageContent) to smaller methods/hooks for more readable code, instead of increasing the limit?

Reply:
well, i face this problem already second time in the same file. its not really about splitting but about the number of logical operators. it does not matter if you move your code to utilities and have couple of extra || or && and still facing the same issue